python/artm/regularizers.py

```diff
@@ -115,18 +104,17 @@ def config(self):
     def type(self):
         return self._type
 
-    @topic_names.setter
-    def topic_names(self, topic_names):
-        _reconfigure_field(self, topic_names, 'topic_names')
+    @config.setter
```

Comment:
``@config.setter`` should invoke ``master_reconfigure()``, to be consistent with other setters